js/private/js_run_devserver.mjs

```diff
@@ -139,6 +160,21 @@ async function syncRecursive(src, dst, writePerm) {
 async function sync(files, sandbox, writePerm) {
     console.error('Syncing...')
     const startTime = perf_hooks.performance.now()
+    const isWindows = process.platform === 'win32'
+    if (isWindows) {
+        // special handling on Windows for node_modules
+        files = files.map((file) => {
+            const nodeModulesIndex = file.indexOf('/node_modules/')
```

Comment:
Are nested `node_modules` not possible? I would have assumed the `if/else` would be the opposite order...